docs/docker-integration/index.rst

```diff
@@ -0,0 +1,92 @@
+.. _docker-integration:
+
+======================
+Docker, Swarm, Compose
```

Comment:
Reworded title. Left directory name as is since this would require a lot of fixing up to references elsewhere. Deferred that to a follow up. CC @davidoliverSP2